brian2/equations/codestrings.py

```diff
@@ -29,18 +31,31 @@ class CodeString(object):
 
     def __init__(self, code):
 
-        # : The code string
-        self.code = code
+        self._code = code
 
         # : Set of identifiers in the code string
         self.identifiers = get_identifiers(code)
 
+    code = property(lambda self: self._code,
+                    doc='The code string')
```

Comment:
It's just to make the immutability of the `CodeString` very clear: `code` is a read-only attribute and should never be changed. We use this idiom in a few places and it seemed to make sense to use here as well.